doc/cephfs/add-remove-mds.rst

```diff
@@ -1,3 +1,10 @@
+.. note::
+   It is highly recommended to use :doc:`/cephadm/index` or another Ceph
+   orchestrator for setting up the ceph cluster. Use this approach only if you
```

Comment:
s/ceph/Ceph/ two instances

Reply:
Making a change here would also mean I need to change it in the master commit too but I'ts already merged

Reply:
I _think_ it's okay to make the change and then add mention about the change in commit message.